src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    protected Color color;
+    protected double area;

```

Comment:
do we need area field?

Reply:
not fixed. Delete this variable. The area should be calculated in the method